galley/pkg/crd/validation/webhook.go

```diff
@@ -126,13 +124,15 @@ type Webhook struct {
 	ownerRefs            []v1.OwnerReference
 	webhookConfiguration *v1beta1.ValidatingWebhookConfiguration
 	endpointReadyOnce    bool
-	monitoringPort       uint
 }
 
 // NewWebhook creates a new instance of the admission webhook controller.
 func NewWebhook(p WebhookParameters) (*Webhook, error) {
+	fmt.Println(p.CertFile)
+	fmt.Println(p.KeyFile)
```

Comment:
Please remove before the final check-in :)

Reply:
done